src/components/ImportRecordsFlow/helpers/__test__/getDataFromWorkbook.spec.js

```diff
@@ -0,0 +1,15 @@
+import XLSX from 'xlsx';
+import getArrayDataFromWorkBook from '../getDataFromWorkbook';
```

Comment:
we should import with the same name `getDataFromWorkbook` in order to avoid confusion